packages/template-retail-react-app/app/pages/account/wishlist/index.jsx

```diff
@@ -107,7 +107,8 @@ const AccountWishlist = () => {
         return isValidChange
     }
 
-    const isPageLoading = wishListItems ? isProductsLoading : isWishListLoading
+    const hasWishlistItems = wishListItems?.length > 0
+    const isPageLoading = hasWishlistItems ? isProductsLoading : isWishListLoading
```

Comment:
The old condition only worked when wishListItems was null. This fix accounts for when wishListItems exists but is empty.

Reply:
Can we add a test to verify this scenario?